src/pages/GetStarted/__tests__/GetStarted.spec.tsx

```diff
@@ -1,89 +0,0 @@
-/*
```

Comment:
Why did you delete this test suit?

Reply:
It was accidentally during rebase. I have returned this code